src/backend/marsha/e2e/test_lti.py

```diff
@@ -0,0 +1,378 @@
+    lti_select_iframe.click(f'[title="Select {document.title}"]')
+    lti_select_iframe.wait_for_selector("dd")
+    assert document_content_items in lti_select_iframe.content()
+
+    # Select a video
+    page.click('#lti_select input[type="submit"]')
+    lti_select_iframe.click('button[role="tab"]:has-text("Videos")')
+    video_content_items = (
+        escape(
+            json.dumps(
+                {
+                    "@context": "http://purl.imsglobal.org/ctx/lti/v1/ContentItem",
+                    "@graph": [
+                        {
+                            "@type": "ContentItem",
+                            "url": f"{live_server}/lti/videos/{video.id}",
+                            "title": f"{video.title}",
+                            "frame": [],
+                        }
+                    ],
+                }
+            )
+        )
+        .replace(", ", ",")
+        .replace(": ", ":")
+    )
+    assert video_content_items not in lti_select_iframe.content()
+    lti_select_iframe.click(f'[title="Select {video.title}"]')
+    assert video_content_items in lti_select_iframe.content()
+    assert Video.objects.count() == 1
+
+
+@pytest.mark.django_db()
+@override_settings(DEBUG=True, BYPASS_AWS_S3=True, X_FRAME_OPTIONS="")
+def test_lti_video_play(page: Page, live_server: LiveServer, mocker):
+    """Test LTI Video play."""
+    page.set_viewport_size({"width": 1200, "height": 1200})
+
+    resolutions = [480]
+    video = VideoFactory(
+        uploaded_on=timezone.now(),
+        resolutions=resolutions,
+    )
+    mock_lti_view = mocker.patch.object(
+        marsha.core.serializers.VideoBaseSerializer, "get_urls", autospec=True
+    )
+
+    mock_lti_view.return_value = {
+        "manifests": {
+            "hls": f"{live_server.url}/static/e2e/hls",
+        },
+        "mp4": {
+            "480": f"{live_server.url}/static/e2e/big_buck_bunny_480p.mp4",
+        },
+        "thumbnails": {
+            "480": f"{live_server.url}/static/e2e/big_buck_bunny_480p.jpg",
+        },
+    }
+
+    lti_consumer_parameters = {
+        "uuid": str(video.id),
+        "resource_link_id": "example.com-df7",
+        "context_id": video.playlist.lti_id,
+        "roles": random.choice(["instructor", "administrator"]),
+        "resource": "videos",
+        "user_id": "56255f3807599c377bf0e5bf072359fd",
+        "lis_person_contact_email_primary": "contact@openfun.fr",
+        "custom_component_display_name": "LTI Consumer",
+        "lti_version": "LTI-1p0",
+        "lis_person_sourcedid": "John",
+        "lti_message_type": "basic-lti-launch-request",
+        "launch_presentation_return_url": "",
+        "lis_result_sourcedid": "course-v1%3Aufr%2Bmathematics%2B0001:"
+        "example.com-df7b0f2886f04b279854585735a402c4:"
+        "56255f3807599c377bf0e5bf072359fd",
+        "launch_presentation_locale": "en",
+    }
+
+    lti_parameters, passport = generate_passport_and_signed_lti_parameters(
```

Comment:
```suggestion
    lti_parameters, _ = generate_passport_and_signed_lti_parameters(
```

Reply:
Or better:
```suggestion
    lti_parameters, _passport= generate_passport_and_signed_lti_parameters(
```

To avoid clashing with the usual meaning of "_" in Django (lazy translation)